Style/index.css

```diff
@@ -1,5 +1,52 @@
-#divBody{
-	padding-left: 20%;
-	padding-right: 20%;
-	padding-top: 2%;
+* {
+	margin: 0;
+	padding: 0;
+}
+
+h1 {
+	padding: 1em;
+}
+
+h1 img {
+	width: 100%;
+	margin: auto;
+	display: block;
+	max-width: 800px;
+}
+
+menu {
+	display: block;
+	
+	font-size: 14px;
+	text-align: center;
+	background-color: #8BC3D6;
+	font-family: Tahoma, Geneva, sans-serif;
+}
+
+menu li {
+	padding: 0.75em;
+	display: inline-block;
+}
+
+menu a {
+	color: inherit;
+	text-decoration: none;
+}
+
+menu li:hover {
```

Comment:
I fixed that annoying thing with the hover where it wouldn't fill all of the menu.